theSrc/scripts/buildConfig.js

```diff
@@ -296,14 +297,13 @@ function buildConfig (userConfig, width, height) {
         .map(z => 2 * LegendUtils.normalizedZtoRadius(config.pointRadius, z))
   }
 
-  if (config.zTitle) {
-    if (config.legendBubbleTitleWrap && config.legendBubbleTitleWrapNChar) {
-      config.legendBubbleTitle = wrapByNumberOfCharacters(config.zTitle, config.legendBubbleTitleWrapNChar).split('<br>')
-    } else {
-      config.legendBubbleTitle = [ config.zTitle ]
-    }
+  if (!config.legendBubbleTitle) {
+    config.legendBubbleTitle = config.zTitle ? config.zTitle : ''
```

Comment:
Does `legendBubbleTitle` need to be an array? Also, I think you can use `??` here.  i.e. `[config.zTitle ?? '']`.

Reply:
It needs an array, because that's the expected output after wrapping. I also tried using `??`, but it gives me a syntax error, and won't build

Reply:
But at line 303, I would need another check for whether or not its already an array?

Reply:
If `zTitle` is not empty, it still needs to be wrapped

Reply:
Ah OK I see. Ideally the type should not change. Maybe a local variable could be used instead to store the title before it is turned into an array.